File: _solutions/05_live/05_live.R
library(ellmer)

# Your job: work with a chatbot to roast Hadley Wickham.
chat <- chat_openai()

# Converse with the chatbot in your console.
live_console(chat)

# After a bit, exit the chat and try chatting in a Shiny app.
live_browser(chat)

File: _solutions/05_live/05_live.py
# %%
import chatlas
import dotenv

dotenv.load_dotenv()


# %% [markdown]
# Your job: work with a chatbot to roast Hadley Wickham.

# %%
chat = chatlas.ChatOpenAI()

# %% [markdown]
# Converse with the chatbot in your console.

# %%
chat.console()

# %% [markdown]
# After a bit, exit the chat and try chatting in a Shiny app.

# %%
chat.app()
